Dockerfiles/manifests/stackstate-agent.yaml

```diff
@@ -0,0 +1,86 @@
+apiVersion: extensions/v1beta1
```

Comment:
According to docs  https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/
daemonSet has left beta state, and now is under apiVersion: apps/v1